### PR TITLE
Introduce SharedResource Localisation to ControlPanel

### PR DESCRIPTION
**Pull request by leigh-pointer**
Updated the ControlPanel to use Shared resources for local strings ;

 **Add**, **Edit**, **Delete**, **Cancel**

Corrected the SharedResources.cs namespace.

### CODE REVIEW

File: Oqtane.Client/SharedResources.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-namespace Oqtane.Client
+namespace Oqtane
```
Comment (hishamco) [inline Oqtane.Client/SharedResources.cs:1]: Why?

**Review by hishamco — COMMENTED** [review]
Just a comment for almost your PRs, why I see extract commits that are not related. Please make your local dev branch up to date before you create a PR

Thanks

**Comment by hishamco** [comment]
Could you apply the change in all the places in the control panel

File: Oqtane.Client/SharedResources.cs (hunk 1 above)
Comment (hishamco) [reply Oqtane.Client/SharedResources.cs:1]: > Corrected the SharedResources.cs namespace.

I know ;) but why, while the class in `Oqtane.Client`

**Comment by sbwalker** [comment]
@leigh-pointer there is someone who is going through the entire Oqtane.Client project this week to ensure all localizations are up to date ( see item 2 on https://github.com/oqtane/oqtane.translations/issues/34 ) - it would be helpful to not introduce PRs at this moment as it will cause conflicts when the large PR is submitted.

**Comment by sbwalker** [comment]
@hishamco there are not any other items of static text in the control panel which belong in the SharedResources. SharedResources should only be used for common terms which are used in multiple areas of the user interface - all unique terms to the razor component should be localized in its own dedicated resx file.

**Comment by hishamco** [comment]
> there is someone who is going through the entire Oqtane.Client project this week to ensure all localizations are up to date

Who is ;)

> SharedResources should only be used for common terms which are used in multiple areas of the user interface

But those terms can be found in many places

**Pull request merged**